tests/psa-client-server/psasim/tools/psa_autogen.py

```diff
@@ -15,6 +15,9 @@
 
 FILENAME = str(sys.argv[1])
 
+SCRIPT_PATH = os.path.dirname(__file__)
```

Comment:
Small side-note. We have a [tool](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/mbedtls_dev/build_tree.py#L78) that allows you to find the absolute path. 

    from mbedtls_dev import build_tree
    root_dir = build_tree.guess_mbedtls_root()